### PR TITLE
M1: Substrate Tokenomics Survey

### PR DESCRIPTION
**Pull request by taqtiqa-mark**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [ ] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH/fiat account provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/1348

### CODE REVIEW

**Comment by semuelle** [comment]
Thanks for the delivery, @taqtiqa-mark. We will look into it as soon as possible!

File: deliveries/tokeneconomics-survey-2022_1.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account that submitted the application is allowed to submit milestones.
+>
+> Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).**
+
+* **Application Document:** [Tokeneconomics Survey 2022](https://github.com/w3f/Grants-Program/tree/master/applications/tokeneconomics-survey-2022.md).
```
Comment (semuelle) [inline deliveries/tokeneconomics-survey-2022_1.md:9]:
```suggestion
* **Application Document:** [Tokeneconomics Survey 2022](https://github.com/w3f/Grants-Program/blob/master/applications/tokenomics-survey-2022.md).
```

**Review by takahser — CHANGES_REQUESTED** [review]
@taqtiqa-mark thanks for your patience. I took a look and added my [interim evaluation](https://github.com/w3f/Grant-Milestone-Delivery/blob/531df91b38f8d02fc3b9285a201d84cc1d8df78f/evaluations/tokenomics-survey-2022_1_takahser.md). Feel free to have a look and let me know, in case you have any questions.

**Comment by taqtiqa-mark** [comment]
Thanks for the prompt review and thoughtful feedback. Please note the W3F changed the required template partway through this application, and in response to this application.  I believe some of the issue raised (e.g. license) reflect the fact this is its first use.
I believe the following address the issues raised:

- 0a. | Copyright and Licenses: This is per the deliverables
- 0b. | Documentation/Tutorial: From the W3F research template:
> Ideally, we recommend the usage of LaTeX/Overleaf for article production and Docker files for software execution.
- 0d. Infrastructure:  The W3F template can (in future) link to the Overleaf offline instructions, rather then require everyone to duplicate this effort.  See 0b
- 0e.2  Section: Polkadot Parachains: Thanks for the feedback. This is a first draft and doesn't seem reasonable to block acceptance of the milestone delivery.

**Comment by takahser** [comment]
@taqtiqa-mark, thank you for the feedback. You're right about _0a._, _0b._, and _0d._. I wasn't aware that these were recommended in the template. As for _0e.2_, I understand that it's a work in progress, and I look forward to your expansion on it in future milestones. Please keep in mind that for finalized chapters, higher standards will apply during the evaluation process for future deliveries.

**Comment by takahser** [comment]
@taqtiqa-mark I can't find the invoice for this milestone. Have you submitted it?

**Comment by taqtiqa-mark** [comment]
Unless I can invoice as an individual .... 

The invoice may take a while - my accountant is reorganizing the company to be used and this may take some time.  I wanted to deliver the milestones, and am happy to continue and defer payment until the company details are in hand.

Apologies for any inconvenience.

**Comment by takahser** [comment]
@taqtiqa-mark okay that works, no problem. Just add a comment here and tag me once the invoice is ready so I can forward it to our finance department.

**Comment by semuelle** [comment]
Hi @taqtiqa-mark, our finance team would like to wrap up 2023, so I would like to ask you to submit an invoice soon. If you have to change the benefiting entity of the grant, feel free to submit an amendment.

**Comment by takahser** [comment]
@taqtiqa-mark reminder that the invoice still hasn't been filed. Until when can we expect it to be submitted?

**Comment by taqtiqa-mark** [comment]
Apologies, this is still being setup. Progress has been made, but not completed. This is not is my hands but our tax year ends 30 June, and I hope this serves as a hard deadline.

**Comment by takahser** [comment]
@taqtiqa-mark understood, thanks for the feedback.

**Comment by taqtiqa-mark** [comment]
@takahser I've submitted the invoice for the 3 milestones. Thanks for your patience.

**Comment by takahser** [comment]
@taqtiqa-mark thanks, it will take up to 2 weeks to process the invoice and initiate the payout.

**Pull request merged**